Altalogy/tari
# feat: open terminal

Description
---

- [x] Open terminal
- [x] Adjust Footer text (it always works on the `ctrl/cmd + T` shortcut

Motivation and Context
---

#29 

How Has This Been Tested?
---

Tested on Windows, Ubuntu and MacOS:

1. Start the application (development)
2. Press `Ctrl + T` (on MacOS use `Cmd + T`)
3. The Terminal should be open

MacOS:

https://user-images.githubusercontent.com/11715931/178327002-d4db9bee-8520-4d03-8a2f-e89781a79974.mp4
